### ISSUE TITLE
508-defect-2 [COGNITION]: BAM2 Debt Letters - Heading level should only increase by one

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16783]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Jennifer_

## User Story or Problem Statement

As a screen reader user, I would like to hear headings in a proper nesting order according to their importance. When I hear heading levels skipped, it makes me wonder if I missed some important content.

## Details

The axe browser plugin is reporting a few heading levels being nested improperly. Screen reader users overwhelmingly use headings to navigate through pages, so nesting these appropriately has a big human impact. I've included a few screenshots with heading levels noted and recommend the team check out a [Chrome plugin called headingsMap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) to see the same left pane I'm showing in the screen shots.

## Acceptance Criteria

- [ ] Heading level best practice issues no longer appear in the axe browser plugin (footer H4 best practice items may still appear, but that's an issue outside the scope of this ticket)
- [ ] No visual changes noted
- [ ] Screen readers announce heading level nesting appropriately

## WCAG or Vendor Guidance (optional)

* https://dequeuniversity.com/rules/axe/4.0/heading-order

## Screenshots or Trace Logs

![Screen Shot 2020-12-01 at 4 19 44 PM](https://user-images.githubusercontent.com/934879/100804518-a3508f00-33f2-11eb-9184-e6d4111e3b1e.png)

---
![Screen Shot 2020-12-01 at 4 09 40 PM](https://user-images.githubusercontent.com/934879/100804536-ab103380-33f2-11eb-86a2-499857244c79.png)

---
![Screen Shot 2020-12-01 at 4 14 00 PM](https://user-images.githubusercontent.com/934879/100804546-b2374180-33f2-11eb-8791-4175ffac906d.png)

**Comment by bradl3yC** [comment]
@1Copenut I do not see any violations with Axe plugin for heading levels - the only one I see based on your screenshots using the lefthand number as a heading level indicator and their nesting, is the H4 in the final screenshot.

**Comment by 1Copenut** [comment]
@bradl3yC The H4 is the only item that'll show up on the axe browser plugin. It'll be a Best Practice item. I flagged the others to be retagged for users who have visual acuity and use assistive technology. Having a visually equal heading as an H2 further up the page and an H3 further down the page is potentially confusing. 

This is a high impact item to assistive technology users, but if it's difficult to adjust, please pivot to other high-priority items and circle back to this one.

**Comment by bradl3yC** [comment]
@1Copenut whats the fix here? should all of the heading levels be the same or decreased in size down the page? If so, we should get @rileyorr and @rebeccawalsh to weigh in

**Comment by rebeccawalsh** [comment]
@bradl3yC Looks like he's included his recommendations in the annotated screenshots above. I believe we pulled most of this styling from other parts of the site - @1Copenut do you suggest using the default heading styling for each level, or using utility classes to mimic the current visual experience?

**Comment by bradl3yC** [comment]
@rebeccawalsh I understand now - opening a PR now

**Comment by bradl3yC** [comment]
@1Copenut this is merged and should be in staging for validation in the next hour or so

**Comment by 1Copenut** [comment]
@bradl3yC 3 small adjustments and we're good to go:

![Screen Shot 2020-12-14 at 5 23 51 PM](https://user-images.githubusercontent.com/934879/102148268-cab34d00-3e31-11eb-8d0d-8ca614c3440a.png)

---
![Screen Shot 2020-12-14 at 5 25 04 PM](https://user-images.githubusercontent.com/934879/102148302-d3a41e80-3e31-11eb-9458-21ceab773193.png)

---
![Screen Shot 2020-12-14 at 5 20 09 PM](https://user-images.githubusercontent.com/934879/102148324-da329600-3e31-11eb-9516-5ce76f987c1f.png)

**Comment by bradl3yC** [comment]
@1Copenut I made the adjustments you requested

**Comment by 1Copenut** [comment]
Looks on point! Thank you @bradl3yC. Moving this one to close.
